src/components/dialogmote/endre/EndreDialogmoteTekster.tsx

```diff
@@ -69,12 +83,23 @@ const EndreDialogmoteTekster = ({ dialogmote }: Props) => {
           generateArbeidstakerTidStedDocument(values)
         }
       />
-      <FritekstSeksjon
-        fieldName="begrunnelseArbeidsgiver"
-        label={texts.begrunnelseArbeidsgiver}
-        handlePreviewClick={() => setDisplayEndringArbeidsgiverPreview(true)}
-        maxLength={MAX_LENGTH_ENDRE_BEGRUNNELSE}
-      />
+      {noNarmesteLeder ? (
+        <FritekstSeksjon
+          fieldName="begrunnelseArbeidsgiver"
+          label={texts.begrunnelseArbeidsgiver}
+          handlePreviewClick={() => setDisplayEndringArbeidsgiverPreview(true)}
+          maxLength={MAX_LENGTH_ENDRE_BEGRUNNELSE}
+          alertText={texts.noNarmesteleder}
```

Comment:
Kan evt gjøre `alertText={noNarmesteLeder && texts.noNarmesteleder}` så slipper man gjenta FritekstSeksjon-en

Reply:
Dette evaluerer til en boolean, og det skal jo være en tekst. Men vi jobba videre med idéen og brukte en ternary for å slippe å gjenta FritekstSeksjonen.

Reply:
Bør fungere med det boolske uttrykket også, ref: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator, men ternary er like bra 👍

Reply:
I den du lenker, ser det ut til at `&&` kun funker med komponenter?

```
  <div>
    {boolskuttykk && <komponent />}
  </div>
```

Så det kanskje ikke funker for en komponents prop 🤔 

```
<Komponent 
  detteErIkkeLov={bool && "ja"}
  menDetteFunker={bool ? "ja" : "nei"}
/>
```

Reply:
Ah, ja, stemmer det!